backend/src/services/course/course.service.ts

```diff
@@ -94,8 +97,15 @@ class Course {
 
   public getAllCoursesStudying(
     userId: number,
-  ): Promise<CourseGetResponseDto[]> {
-    return this.#courseRepository.getAllCoursesStudying(userId);
+    pagination: EntityPaginationRequestQueryDto,
+  ): Promise<EntityPagination<CourseGetResponseDto>> {
+    const { page, count } = pagination;
+    const zeroIndexPage = page - 1;
```

Comment:
let's introduce helper for this
![image](https://user-images.githubusercontent.com/69893643/191028073-e4334923-fb94-4377-a587-378e3bf0921c.png)